src/map/status_effect_container.cpp

```diff
@@ -1313,6 +1300,49 @@ void CStatusEffectContainer::UpdateStatusIcons()
     }
 }
 
+uint8 CStatusEffectContainer::GetStatusEffectCountInRange(EFFECT start, EFFECT end)
+{
+    uint8 count = 0;
+    for (CStatusEffect* PStatusEffect : m_StatusEffectSet)
+    {
+        if (PStatusEffect->GetStatusID() >= start && PStatusEffect->GetStatusID() <= end && !PStatusEffect->deleted)
+        {
+            count++;
+        }
+    }
+    return count;
+}
+
+void CStatusEffectContainer::RemoveOldestStatusEffectInRange(EFFECT start, EFFECT end)
```

Comment:
"in range" is terminology we might use fore something distance related - perhaps we could call it RemoveOldestStatusEffectByID since its an id range, not a distance range? Or Any naming convention you can think of that at a glance tells me this isn't for something distance range related

Reply:
maybe "IdRange" is close enough terminology to keep the average contributor from confusion - what do you think?

Reply:
I agree, I didn't like 'range' as a term for this, 'IDRange' could be better